pkgs/stdenv/generic/default.nix

```diff
@@ -54,7 +60,11 @@ let
       pos' = if pos != null then "‘" + pos.file + ":" + toString pos.line + "’" else "«unknown-file»";
 
       throwEvalHelp = unfreeOrBroken: whatIsWrong:
-        assert (unfreeOrBroken == "Unfree" || unfreeOrBroken == "Broken");
+        assert (builtins.elem unfreeOrBroken [ "Unfree"
+                                               "Broken"
+                                               "AllowedLicense"
+                                               "BlacklistedLicense"
```

Comment:
You're not using BlacklistedLicense? Shouldn't AllowedLicense be taken out and instead use BlacklistedLicense?